### PR TITLE
docs: add initial changelog

### PR DESCRIPTION
**Pull request by schwma**

### CODE REVIEW

**Review by chgeo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Version 0.1.0 - tbd
```
Comment (hm23) [inline CHANGELOG.md:8]: tbd -> today (or remove)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- Improved support for aliases on fields that represent compositions and associations (some limitations still apply)
+- Include localized `texts` fields of entities in schema generation
+- Improve check to skip field `localized` during schema generation
+
+### Changed
+
+- The GraphQL protocol adapter now uses a new middlewares mechanism instead of `cds.plugins` which requires `@sap/cds` version 6.3 to run. Enable the `cds.requires.middlewares` flag and register the GraphQL protocol adapter in `cds.env.protocols` to get started.
+- Replaced `debug` level query and mutation logging with improved `info` level request logging
+
+### Fixed
```
Comment (hm23) [inline CHANGELOG.md:35]: remove empty sections

**Review by hm23 — APPROVED** [review]
... apart from the two comments 👍

File: CHANGELOG.md (hunk 1 above)
Comment (schwma) [reply CHANGELOG.md:8]: I thought we would do that in the merge from `main` to the `release` branch, but we could also do it now. What do you think?

Comment (chgeo) [reply CHANGELOG.md:8]: Yes, let's do that when we know the date.

**Pull request merged**